tests/app/celery/test_tasks.py

```diff
@@ -221,3 +221,39 @@ def side_effect():
             call(name="update-letter-job-to-error", args=("3872ce4a-8817-44b9-bca6-972ac6706b59",), queue="notify"),
             call(name="update-letter-job-to-error", args=("c278d13c-40cf-4091-ac88-3ef6eaeae4e8",), queue="notify")
         ]
+
+
+def test_should_not_try_and_send_a_file_if_all_jobs_failed(client, mocker):
+    with freeze_time('2016-01-01T17:00:00'):
+        def side_effect():
+            return \
```

Comment:
i don't like backslash, mainly because i always screw up when i use them, and in cases when it matters (mostly involving bash/makefile) it's easy to forget how they handle whitespace (ie not at all, they just remove the following `\n`)